xml/cl.xml

```diff
@@ -6444,6 +6498,9 @@ server's OpenCL/api-docs repository.
         <extension name="cl_khr_gl_depth_images" requires="cl_khr_gl_sharing" comment="no API - reuses tokens from core API" supported="opencl">
             <require>
                 <type name="CL/cl.h"/>
+                <type name="cl_GLint"/>
+                <type name="cl_GLenum"/>
+                <type name="cl_GLuint"/>
```

Comment:
If we're going to attach these to an extension then I think it makes the most sense to attach them to `cl_khr_gl_sharing` and not to `cl_khr_gl_depth_images`.

Note that these types are also defined in `cl_platform.h`, though that's probably a bug, and I think it'd be pretty low-risk to define them in an extension header (specifically `cl_gl.h`) instead.

Reply:
Agree, which, looking at my commit, message seemed to have been the intent.

Also agree, I've created https://github.com/KhronosGroup/OpenCL-Headers/pull/257

Reply:
The other PRs have gone in so I'll resolve this thread. I've rebased this change again, over to you for when you get a chance to look at the other threads on this PR.